assets/javascripts/render.js

```diff
@@ -198,5 +200,12 @@ function renderModuleTable(container, response) {
         }
 
         tbody.appendChild(renderModuleRow(module, response.snippets));
+
+        if (summary[module.result] === undefined)
+            summary[module.result] = 1;
+        else
+            summary[module.result] ++;
     }
+
+    $('#summary_stars').html(renderTestSummary(summary, 'display'));
```

Comment:
After you've moved the summary to the info box, none of this code should be within the `renderModuleTable()` function anymore. This function is *only* about the contents of the details tab. However, the summary should be shown in any case. And yes, that might mean that you need to stash the job module statistics when rendering the "main page". However, that shouldn't be problematic because you actually don't have to sum up anything within a loop - these figures are directly available as columns of the jobs table (e.g. `passed_module_count`) and it would be the simplest approach to make use of these columns anyways.

Note that any JavaScript code you need to run to initialize the info panel (you could keep using the `renderTestSummary()` function) needs to be re-run in `refreshInfoPanel()` which is triggered when the job state changes. (For testing purposes you can of course simply invoke this function from the JavaScript console.)

Reply:
Please, review the alternative in https://github.com/os-autoinst/openQA/pull/3222